roles/provision-v2v/tasks/main.yml

```diff
@@ -1,4 +1,4 @@
 - include: "login.yml"
 
 - name: execute run script
-  shell: ./run.sh {{ vm_name }} {{ username }} {{ password }} {{ url }}
+  shell: ./run.sh {{ vm_name }} {{ username }} {{ password }} {{ url }} {{ os_type }}
```

Comment:
Is the os_type propagated to OVM, i.e. via label?

Reply:
Yes, please see: https://github.com/kubevirt/v2v-job/pull/7